hack/run-user-cluster-controller-manager.sh

```diff
@@ -85,6 +85,7 @@ set -x
   -log-debug=$KUBERMATIC_DEBUG \
   -log-format=Console \
   -logtostderr \
+  -tunneling-agent-ip=192.168.30.10 \
```

Comment:
Because I'm a noob: Where does this IP come from? Could I also use `192.168.30.9`?

Reply:
seems to be hard-coded in the seed-controller-manager: https://github.com/kubermatic/kubermatic/blob/3a93c3489da1d3c5249edac40a6cb749d4a7f296/cmd/seed-controller-manager/options.go#L122-L123

Reply:
I think it in theory can be any IP from an IP range that does not collide with node subnet / pod CIDR / services CIDR? (cc @youssefazrak )

Reply:
It is indeed hardcoded. This is what we chose with Iacopo for the tech preview and the goal is to make this one configurable as of right now, there is a probability of colliding with an existing IP in the VPC, data center, whatever is called.
Regarding the `192.168.30.10` choice it is random, in one of the 3 private ranges, and not one of the used ones by default like `192.168.0.0/24`.

Reply:
> right now, there is a probability of colliding with an existing IP in the VPC, data center

Does this affect production usage or only when 2 people use the script at the same time for debugging stuff?

Reply:
@xrstf it could affect production. Depending on the networks tbh.
In all cases, this should be made configurable.